Sources/Shared/ViewModel.swift

```diff
@@ -93,6 +100,7 @@ public struct ViewModel: Mappable {
   */
   public init(_ map: JSONDictionary) {
     index    <- map.property(.Index)
+    identifier = map.property(.Identifier)
```

Comment:
The compiler threw a bunch of complaints when trying to assign a value to something that is yet to be initialized using that operator.